app/controllers/declaration/ItemsSummaryController.scala

```diff
@@ -69,24 +70,30 @@ class ItemsSummaryController @Inject() (
 
   // TODO Should we add validation for POST without items?
   def submit(): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
-    val incorrectItems: Seq[FormError] = buildIncorrectItemsErrors(request)
-
-    itemSummaryForm
-      .bindFromRequest()
-      .fold(
-        formWithErrors => Future.successful(BadRequest(itemsSummaryPage(formWithErrors, request.cacheModel.items.toList, incorrectItems))),
-        validYesNo =>
-          validYesNo.answer match {
-            case YesNoAnswers.yes =>
-              createNewItemInCache.map(itemId => navigator.continueTo(ProcedureCodesController.displayPage(itemId)))
-
-            case YesNoAnswers.no if incorrectItems.nonEmpty =>
-              Future.successful(BadRequest(itemsSummaryPage(itemSummaryForm.fill(validYesNo), request.cacheModel.items.toList, incorrectItems)))
-
-            case YesNoAnswers.no =>
-              Future.successful(navigator.continueTo(TransportLeavingTheBorderController.displayPage))
-          }
-      )
+    request.declarationType match {
+      case CLEARANCE =>
+        itemSummaryForm.fill(YesNoAnswer(YesNoAnswers.no))
```

Comment:
What's this statement for? Is it actually useful?

Reply:
The form is not displayed yet a submission of the form is still required to move to the next page. I could either bypass validation for a 'Clearance' journey or submit a 'No' form; the latter seemed a safer option.